ui/qt-lib/resources/stylesheet.qss

```diff
@@ -346,6 +346,27 @@ QDialog QPushButton[styleRole="passwordVisibilityToggle"]:checked {
     background: #fafafa;
 }
 
+#sendDialog QComboBox {
```

Comment:
Huh?

Reply:
That was part for fees type selector, which we desided to remove from code cause it doesn't implemented in backend yet:)

Reply:
Please bring these styles in correspondence with `QComboBox` stuff in NewWalletDialog. I suppose they should be merged. Since AD-393 is merged, those styles should be correct everywhere.